js/common.js

```diff
@@ -121,13 +123,13 @@ function showBuildInfo(buildRevision) {
             "limit":1,
             "format":"list",
             "select":["build.revision","build.created_timestamp","build.taskId"],
-            "from":"coverage",
+            "from":"coverage-summary",
             "where":{"eq":{"build.revision":buildRevision}}
         }));
 
         var taskId = result.data[0].build.taskId;
-        var url = `https://tools.taskcluster.net/task-inspector/#${taskId}/`;
```

Comment:
This is [string interpolation](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals) aka Template literals in ES6